pkgs/applications/networking/instant-messengers/teams/default.nix

```diff
@@ -0,0 +1,124 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, wrapGAppsHook
+, dpkg
+, libuuid
+, nspr
+, nss
+, alsaLib
+, pulseaudio
+, cups
+, expat
+, dbus
+, glib
+, cairo
+, pango
+, atk
+, at-spi2-atk
+, gtk3
+, gdk-pixbuf
+, libxcb
+, libxkbfile
+, libX11
+, libXext
+, libXfixes
+, libXi
+, libXcursor
+, libXcomposite
+, libXdamage
+, libXrender
+, libXrandr
+, libXtst
+, libXScrnSaver
+, libsecret
+, libgnome-keyring
+, coreutils
+, gawk
+, xdg_utils
+, systemd }:
+
+assert stdenv.hostPlatform.system == "x86_64-linux";
+
+let
+
+  version = "1.2.00.32451";
+
+in
+
+stdenv.mkDerivation {
+  pname = "teams";
+  inherit version;
+
+  src = fetchurl {
+    url = "https://packages.microsoft.com/repos/ms-teams/pool/main/t/teams/teams_${version}_amd64.deb";
+    sha256 = "1p053kg5qksr78v2h7cxia5mb9kzgfwm6n99x579vfx48kka1n18";
+  };
+
+  nativeBuildInputs = [ dpkg autoPatchelfHook wrapGAppsHook ];
+
+  unpackCmd = "dpkg -x $curSrc .";
+
+  buildInputs = [
```

Comment:
can we just reuse `atomEnv.packages`? We probably need to move that into a better attribute.